Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1864,6 +1877,9 @@
 #if ENABLE(BLOCK_SET_EXCEPTION_PORTS)
             (with-filter (require-not (webcontent-process-launched))
                 (allow mach-message-send (kernel-mig-routine thread_set_exception_ports)))
```

Comment:
You may have to remove the last closing parenthesis here.

Reply:
Should `thread_set_exception_ports` only be allowed when `HAVE(MACH_HARDENED_EXCEPTIONS)` is not true?